fitbenchmarking/results_processing/tests/expected_results/minimizer_dropdown.html

```diff
@@ -0,0 +1,7 @@
+<div id="minimizer_dropdown" class="dropdown-check-list" tabindex="100">
+    <span class="anchor" onclick="show_dropdown('minimizer_dropdown')">Select Minimizers</span>
+    <ul class="items">
+        <li><label class="noselect"><input type="checkbox" checked=true onclick="toggle_minimizer('scipy-ls', 'dogbox')"/> dogbox</label></li>
+        <li><label class="noselect"><input type="checkbox" checked=true onclick="toggle_minimizer('scipy-ls', 'lm-scipy')"/> lm-scipy</label></li>
+        <li><label class="noselect"><input type="checkbox" checked=true onclick="toggle_minimizer('scipy-ls', 'trf')"/> trf</label></li>    </ul>
```

Comment:
It looks like a new line gets missed before the `<\ul>` either that or we should remove the spaces.

Reply:
Nice spot 👍